src/subscript/casegen_upcars/model.py

```diff
@@ -92,6 +92,8 @@ def __init__(
 
         if isinstance(streak_rect, list) and len(streak_rect) > 0:
             if isinstance(streak_rect[0], list) and len(streak_rect) > 1:
+                print(len(streak_rect))
```

Comment:
leftover debugging statements?

Reply:
@berland Good catch. Yes, it is. I will remove it now.